README.md

```diff
@@ -3,6 +3,12 @@ The developer community page of Karakun AG
 
 ## Build instructions
 
-In order to start `start.sh`  you have to install _jekyll_ (see instructions online).
+In order to start `start.sh`  you have to install [_jekyll_](https://jekyllrb.com) (see instructions online).
+
+If you want to install [_jekyll_](https://jekyllrb.com) on macOS,
+you need to execute these commands in your terminal:
+
+$ brew install ruby
```

Comment:
layout the 2 lines as code. See https://github.com/canoo/dolphin-platform for example.